pkg/controller/packet/packet.go

```diff
@@ -27,7 +27,7 @@ type Controllers struct{}
 // SetupWithManager adds all Packet controllers to the manager.
 func (c *Controllers) SetupWithManager(mgr ctrl.Manager) error {
 
-	if err := (&device.Controller{}).SetupWithManager(mgr); err != nil {
+	if err := (&device.DeviceController{}).SetupWithManager(mgr); err != nil {
```

Comment:
This naming is a bit redundant (`device.DeviceController`) as I am not sure there will be another controller in the device package, but I am totally fine with it if you feel it is best :)

Reply:
We could go with `PacketDeviceController` if you think it is better.

Reply:
I was more suggesting just keeping `device.Controller`, but I prefer `device.DeviceController` over `device.PacketDeviceController`.